tests/tls/run.sh

```diff
@@ -47,9 +47,114 @@ function prepare_data() {
 	done
 }
 
-function run() {
+function setup_mysql_tls() {
+	run_sql 'SHOW VARIABLES WHERE Variable_Name = "datadir"' $MYSQL_PORT1 $MYSQL_PASSWORD1
+	mysql_data_path=$(cat "$TEST_DIR/sql_res.$TEST_NAME.txt" | grep Value | cut -d ':' -f 2 | xargs)
```

Comment:
what's the purpose of this `xargs`

Reply:
in L55, we want to use ` mysql_ssl_rsa_setup --datadir "$mysql_data_path"` to setup mysql ssl before

`xargs` can make $mysql_data_path easy to pass on